src/plugins/intel_gpu/src/graph/graph_optimizer/mark_runtime_skippable_nodes.cpp

```diff
@@ -54,7 +55,40 @@ void mark_runtime_skippable_nodes::run(program& p) {
                 if (node.have_user_with_type<concatenation>() && node.get_users().size() == 1)
                     return;
                 node.can_be_optimized(true);
-                GPU_DEBUG_TRACE_DETAIL << "[mark_runtime_skippable_nodes] : " << node.id() << "can_be_optimized" << std::endl;
+                GPU_DEBUG_TRACE_DETAIL << "[mark_runtime_skippable_nodes] : " << node.id() << " can_be_optimized" << std::endl;
+            }
+        });
+        program_helpers::do_for_types<strided_slice>(*node, [](strided_slice_node& node){
+            if (node.has_fused_primitives() || node.is_output())
+                return;
+
+            auto impl_params = node.get_kernel_impl_params();
+            auto prim = impl_params->typed_desc<strided_slice>();
+            if (prim->end_mask.empty()
+                || !prim->new_axis_mask.empty()
+                || !prim->shrink_axis_mask.empty()
+                || !prim->ellipsis_mask.empty()) {
+                return;
+            }
+
+            auto begin = prim->begin;
+            auto strides = prim->strides;
+            if (all_zeroes(begin) && all_ones(strides)) {
```

Comment:
Or we can add begin mask is set too ? Because ignoring begin means that it starts from 0.

Reply:
updated